### ISSUE TITLE
Blueprints populate

### ISSUE THREAD

**Issue by Dallas62** [balderdashy/sails-docs#384]
Refer to: http://sailsjs.org/#/documentation/reference/blueprint-api/Populate.html

I don't know if it's an issue from the documentation or blueprints, but when I do:
```
http://localhost:1337/purchase/47/cashier
```

The result looks like:
```
{
    "name": "Dolly",
    "id": 7,
    "createdAt": "2012-05-14T01:21:05.000Z",
    "updatedAt": "2013-01-15T01:18:40.000Z"
}
```

It's the data of 'cashier', but not include in the 'purchase'.

Doesn't respect:
```
If the specified association is plural ("collection"), this action returns the list of associated records as a JSON array of objects. If the specified association is singular ("model"), this action returns the associated record as a JSON object.
```

**Comment by sailsbot** [comment]
Thanks for posting, @Dallas62.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
